core/base/inc/TPoint.h

```diff
@@ -43,6 +43,7 @@ class TPoint {
    TPoint() : fX(0), fY(0) { }
    TPoint(SCoord_t xy) : fX(xy), fY(xy) { }
    TPoint(SCoord_t x, SCoord_t y) : fX(x), fY(y) { }
+   TPoint(const TPoint &p) : fX(p.fX), fY(p.fY) { }
```

Comment:
Instead of adding this, can we remove the assignment operator (relying on the default) and while we're at it also the destructor?